Subtester/Assets/Scenes/Main.unity

```diff
@@ -136,7 +136,7 @@ GameObject:
   - component: {fileID: 53425792}
   - component: {fileID: 53425791}
   m_Layer: 5
-  m_Name: CustomerInfoLabel
+  m_Name: infoLabel
```

Comment:
updated the name of the label since it now shows everything, not just customer info